src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -82,7 +82,7 @@ public string Value
         /// <exception cref="FormatException">
         ///   Text value of this instance is not in an ISO 8601 defined DateTime format.
         /// </exception>
-        public DateTime GetDateTime() => DateTime.ParseExact(_value, "s", CultureInfo.InvariantCulture);
+        public DateTime GetDateTime() => JsonHelpers.TryParseAsISO(_value.AsSpan(), out DateTime value) ? value : throw new FormatException(SR.FormatDateTime);
```

Comment:
Nit: is the `.AsSpan()` necessary?

Reply:
Yes, because of the `BUILDING_INBOX_LIBRARY` thingy. The build failed on .NET Framework because it doesn't have the implicit conversions.

Reply:
Ah. Right.  Ok.  Darn that OOB build.